integration/e2e/api/config-query-range-max-series-disabled.yaml

```diff
@@ -0,0 +1,81 @@
+target: all
+stream_over_http_enabled: true
+
+server:
+  http_listen_port: 3200
+  log_level: debug
+
+query_frontend:
+  search:
+    query_backend_after: 0 # setting these both to 0 will force all range searches to hit the backend
```

Comment:
I think this e2e test is still querying the generators because this needs to be moved to under `metrics:` instead.

Reply:
added another test that targets the querier